Security/src/Test-CVE-2021-34470.ps1

```diff
@@ -65,7 +65,7 @@ if ($ApplyFix) {
         $storageGroupSchemaEntry.Properties["possSuperiors"] | Out-File $OutputFile -Append
     }
 
-    $isSchemaAdmin = $null -ne (whoami /groups | sls "\\Schema Admins\s+Group")
+    $isSchemaAdmin = $null -ne (whoami /groups | sls "\\Schema Admins\s+")
```

Comment:
I am not 100% this works correctly like we think it does. I have seen a few issues with SetupAssist where they are part of the group vs sub groups vs directly added to it. 

Test by created a secondary group (ChildGroup) place your Admin into ChildGroup. Place ChildGroup as a member of "Schema Admins". 

If that doesn't work, should we address that now?